docs/demos/phase1/sprint3/map-filters-layers-and-occurrence-data.md

```diff
@@ -0,0 +1,56 @@
+# React app stories
+
+## Stories covered
+- [74. Add a filters and layers component to the map](https://github.com/icipe-official/vectoratlas-software-code/issues/74)
+- [98. Add and API route to load occurrence data for the map](https://github.com/icipe-official/vectoratlas-software-code/issues/98)
+- [99. Load occurrence data on map](https://github.com/icipe-official/vectoratlas-software-code/issues/99)
+
+## Demo Setup
+1. Check out `feature/load-occurence-data-on-map-#99`
```

Comment:
Don't change this now, but in future demos should never run off a branch. Really, stories should only be demo'd when they are done and so everything should run off main.